core/core.mk

```diff
@@ -49,6 +49,9 @@ endif
 ifneq ($(CFG_TEE_CORE_DEBUG),y)
 cppflags$(sm)  += -DNDEBUG
 endif
+ifeq ($(CFG_BGET_BESTFIT),y)
+cppflags$(sm)  += -DBestFit=1
```

Comment:
Better do it directly in `lib/libutils/isoc/bget.c`, it is the only file that uses the `BestFit` macro.
```C
#ifdef CFG_CORE_BGET_BESTFIT
#define BestFit 1
#endif
```